agent/build.gradle

```diff
@@ -0,0 +1,36 @@
+
```

Comment:
As we decided that the agent will be published in as a beta version we should explicitly set its version (override the common setting)